### DISCUSSION TITLE
资源配额 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1008] [Announcements]
# 资源配额 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/ghippo/user-guide/workspace/quota/

**Comment by liyuerich** [comment]
使工作空间中的用户/用户组具有该集群资源的使用权限，能够使用资源配额前往应用工作台创建命名空间（Namespace）。 

并没有 前往应用工作台创建命名空间（Namespace）页面

**Reply by yitingdc** [reply]
已经创建issue来跟踪：https://gitlab.daocloud.cn/ndx/ghippo/-/issues/631

**Comment by liyuerich** [comment]
在容器管理中将创建的命名空间 ns01 绑定到 ws01
图片中 资源限制 - 目前是 资源限额

**Reply by Michelle951** [reply]
已修改，感谢越姐反馈问题，帮助改进文档站

**Comment by liyuerich** [comment]
能否补充一下资源配额和资源限额之间的关系？

**Reply by yitingdc** [reply]
用 https://gitlab.daocloud.cn/ndx/ghippo/-/issues/633 跟踪
